sympy/integrals/laplace.py

```diff
@@ -1430,7 +1465,8 @@ def inverse_laplace_transform(F, s, t, plane=None, **hints):
     r"""
     Compute the inverse Laplace transform of `F(s)`, defined as
 
-    .. math :: f(t) = \frac{1}{2\pi i} \int_{c-i\infty}^{c+i\infty} e^{st} F(s) \mathrm{d}s,
+    .. math :: f(t) = \frac{1}{2\pi i} \int_{c-i\infty}^{c+i\infty}
+    e^{st} F(s) \mathrm{d}s,
```

Comment:
Be careful about breaking down the lines in docstrings for RST directives

https://output.circle-artifacts.com/output/job/10de5060-a7ee-48b5-8117-486a2b4c2854/artifacts/0/doc/_build/html/modules/integrals/integrals.html#sympy.integrals.transforms.inverse_laplace_transform

This already shows that the math is broken in docstring.
You may need to reformat to indent correctly to the `.. math::` [directive](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#directives)